stage1/init/kvm/hypervisor/hypervisor.go

```diff
@@ -44,4 +46,9 @@ func (hv *KvmHypervisor) InitKernelParams(isDebug bool) {
 			"quiet systemd.log_level=emerg",
 		}...)
 	}
+
+	customKernelParams := os.Getenv("RKT_HYPERVISOR_CUSTOM_KERNEL_PARAMS")
```

Comment:
Nit: I feel like `RKT_HYPERVISOR_EXTRA_KERNEL_PARAMS` would be a better name (more clear arguments do get appended)?